Assets/HoloToolkit/UX/Scripts/UICollection.cs

```diff
@@ -75,13 +75,13 @@ private void Start()
             UpdateLayout();
         }
 
+#if UNITY_EDITOR
         private void Update()
         {
-#if UNITY_EDITOR
```

Comment:
We should probably use `if (Application.isEditor)` instead.

Reply:
Won't we have a lot of unnecessary calls to methods that won't be there otherwise?

Reply:
What if we tag the method as `[Conditional("UNITY_EDITOR")]`

Reply:
The method gets called whether it's in the editor it but anyway cause it's a unity API